package.json

```diff
@@ -5,16 +5,17 @@
   "workspaces": [
     "packages/*",
     "packages/*/example",
-    "scripts/*"
+    "scripts/*",
+    "examples/*"
   ],
   "directories": {
     "example": "examples"
   },
   "scripts": {
-    "bootstrap": "yarn workspaces foreach --all --exclude=. --exclude=packages/cypress/example --topological run build",
+    "bootstrap": "yarn workspaces foreach --all --exclude=. --exclude=examples/create-react-app-typescript --exclude=packages/cypress/example --exclude=packages/node --topological run build",
```

Comment:
I got confused by the `build` script in `packages/node/package.json` because it only prints "Error: no build specified", so I removed it here.